content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md

```diff
@@ -0,0 +1,157 @@
+---
+weight: 40
+title: Release 10.16.0.0
+layout: bundle
+---
+
+### Improvements
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>DTM application is integrated with Gainsight to capture the data.</td>
+<td>CTM-130</td>
+</tr>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>Graceful handling of custom property and asset type deletion from backend is implemented.</td>
```

Comment:
what does "graceful" mean?

Reply:
Basically we are displaying a warning message stating that property not found